### PR TITLE
Mycroft: update configuration.yaml

### PR DESCRIPTION
**Pull request by Tony763**
## Proposed change
To interact with this platform it is necessary to configure also notify with platform mycroft and name. 

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

ref: [https://community.mycroft.ai/t/picroft-and-home-assistant/8126/3](https://community.mycroft.ai/t/picroft-and-home-assistant/8126/3)

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/mycroft.markdown [hunk 1]
```diff
@@ -22,11 +22,26 @@ To use Mycroft in your installation, add the following to your `configuration.ya
 # Example configuration.yaml entry
 mycroft:
   host: 0.0.0.0
+notify:
```
Comment (frenck) [inline source/_integrations/mycroft.markdown]: Let's split this into a separate section on how to set up notifications, and also split the configuration block.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/mycroft.markdown (hunk 1 above)
Comment (Tony763) [reply source/_integrations/mycroft.markdown]: Ok, I will rewrite it.

**Comment by Tony763** [comment]
@frenck  Is this ok?

**Review by frenck — APPROVED** [review]
Thanks, @Tony763!

Sorry for the long wait! 🙏

**Pull request merged**